src/main/java/com/ericsson/ei/subscriptionhandler/SubscriptionValidator.java

```diff
@@ -51,137 +49,113 @@ public void validateSubscription(Subscription subscription) throws SubscriptionV
         if (subscription.getNotificationType().equals("REST_POST")) {
             this.RestPostMediaType(subscription.getRestPostBodyMediaType());
         }
-        // List<Requirement> reqList = subscription.getRequirements();
-        // for (int i=0; i < reqList.size(); i++) {
-        // this.validateJmespath(reqList.get(i).getConditions().get(0).getJmespath());
-        // }
-        LOG.info("Validating of subscription " + subscription.getSubscriptionName() + " finished successfully.");
+        LOGGER.debug("Validating of subscription " + subscription.getSubscriptionName() + " finished successfully.");
     }
 
-    /*
+    /**
      * Validation of subscriptionName parameter Throws
      * SubscriptionValidationException if validation of the parameter fails due to
      * wrong format of parameter.
-     * 
+     *
      * @param subscriptionName
      */
-    public void validateSubscriptionName(String subscriptionName) throws SubscriptionValidationException {
-
+    private void validateSubscriptionName(String subscriptionName) throws SubscriptionValidationException {
```

Comment:
Doesn't the change to "private" function break the unit tests? or goes all Subscriptions Validations Tests via validateSubscription() function?

Reply:
I used powermockito to test validation

Reply:
I want to use the Embedded Mongo in test,, so I keep the code as it is. That works.